pkg/build/strategies/sti/sti.go

```diff
@@ -115,10 +115,12 @@ func (b *STI) Build(config *api.Config) (*api.Result, error) {
 		return nil, err
 	}
 
-	if b.incremental = b.artifacts.Exists(config); b.incremental {
-		glog.V(1).Infof("Existing image for tag %s detected for incremental build", config.Tag)
+	if !config.Incremental {
+		glog.V(1).Infof("Incremental build is disabled. Clean build will be performed")
+	} else if b.incremental = b.artifacts.Exists(config); b.incremental {
+		glog.V(1).Infof("Existing image for tag %s detected. Perform Incremental build with save-artifacts under the %s", config.Tag, b.scriptsURL["save-artifacts"])
```

Comment:
It is, but again at loglevel=2, see [here](https://github.com/openshift/origin/blob/master/pkg/build/builder/sti.go#L102).